dtests/src/test/java/io/snappydata/hydra/connectionPool/SnappyConnectionPoolPrms.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2017 SnappyData, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+package io.snappydata.hydra.connectionPool;
+
+import hydra.BasePrms;
+import hydra.TestConfig;
+
+public class SnappyConnectionPoolPrms extends BasePrms {
+
+  public static Long useTomcatConnPool;
+
+  public static Long useHikariConnPool;
```

Comment:
Remove this parameter and single parameter with generic name and it can take names of connection pool to be used. e.g. useConnPool = ONEOF tomcat hikari FOENO. This will be useful in confs for running with oneof connection pool dynamically.